### ISSUE TITLE
Need to overwrite Output Directory to deploy on Vercel

### ISSUE THREAD

**Issue by vcidst** [thuglabs/create-dapp-solana-nextjs#23]
When I forked and tried to deploy without making any changes, I got the following error

```
Error: The file "/vercel/path0/.next/routes-manifest.json" couldn't be found. This is normally caused by a misconfiguration in your project.

Please check the following, and reach out to support if you cannot resolve the problem:

1. If present, be sure your `build` script in "package.json" calls `next build`.  
2. Navigate to your project's settings in the Vercel dashboard, and verify that the "Build Command" is not overridden, or that it calls `next build`.  
3. Navigate to your project's settings in the Vercel dashboard, and verify that the "Output Directory" is not overridden. Note that `next export` does **not** require you change this setting, even if you customize the `next export` output directory. Learn More: https://err.sh/vercel/vercel/now-next-routes-manifest</span>
```

I had to change the "Output Directory" to "build" in project settings on Vercel dashboard to deploy it successfully. I'm not familiar with next.js ecosystem but I realized this is defined in `next.config.js`.

If this is something non-standard, can we add this bit to the readme?

**Comment by jamland** [comment]
Hi! I suppose this is something related to `vercel` deployment settings rather than this repo code.
What usually do is set the same settings I use locally when settings up new app on vercel. Check `Build and Output Settings` personally I always set it like this and it works for me

![image](https://user-images.githubusercontent.com/188568/154467790-51de90d8-5a72-4cf5-b9e7-070d510bb914.png)

**Comment by jamland** [comment]
if this is helpful I can add it to `README` ofc

**Comment by vcidst** [comment]
Yes, that would be helpful. I wasn't aware of these configurations. I found out the fix from this [Vercel error note](https://github.com/vercel/vercel/blob/main/errors/now-next-routes-manifest.md), specifically the point 2.

These settings worked for me,
![](https://i.imgur.com/eYhcpRA.png)

**Comment by c-ehrlich** [comment]
Could you provide a reproduction (ideally as close to a stock create-t3-app as possible) where it fails? Ideally with as few packages selected as possible, to make it easier to figure out what is causing it.

I just got an unchanged create-t3-app to deploy to Vercel without any problem.

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/8353666/205466455-a86a3396-fbf4-44a3-9da8-a31cccf87e5d.png">

**Comment by johncanero** [comment]
Thank you for this @vcidst !! It worked in my deployment.
